src/toast/toast.styles.tsx

```diff
@@ -89,21 +98,17 @@ export const Description = styled.div<StyleProps>`
 `;
 
 export const CloseIcon = styled(CrossIcon)`
-    margin-top: 0.2rem;
+    margin: 0 !important;
```

Comment:
avoid using `!important`

my suggestion is on line 62, change it to
```
& > svg {
```
to target only the status icons

and reapply the svg height/width to the svg selector within DismissButton

then you can also use `CrossIcon` directly in the toast component